docs/source/kedro_project_setup/configuration.md

```diff
@@ -8,7 +8,10 @@ We recommend that you keep all configuration files in the `conf` directory of a
 ```python
 CONF_SOURCE = "new_conf"
 ```
-
+You can also specify a source directory for the configuration files at run time using the `--conf-source` flag with the `kedro run` command as follows:
```

Comment:
Maybe include a link here to the `commands_reference.md` table?

Reply:
Done! c1b8e2c46974495cce4b0134ab3beb482efdd160